### PR TITLE
setup forklift for individual users

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: centos.org/pipelines/lib/foremanDuffyJob.groovy [hunk 1]
```diff
@@ -66,22 +81,23 @@ pipeline {
...
+                            def boxname = "${username}@duffy_box"
                             branches[name] = {
                                 def playBook = pipelineVars(action: action, type: params.type, version: params.version, os: os, extra_vars: ['expected_version': params.expected_version ?: ''])
-                                def extra_vars = buildExtraVars(extraVars: playBook['extraVars'] + ["forklift_vagrant_lock": "/tmp/forklift.lock"])
-                                def playbooks = duffy_ssh("ls forklift/pipelines/${playBook['pipeline']}", 'duffy_box', './', true)
+                                def extra_vars = buildExtraVars(extraVars: playBook['extraVars'])
+                                def playbooks = duffy_ssh("ls forklift/pipelines/${playBook['pipeline']}", boxname, './', true)
                                 playbooks = playbooks.split("\n")
 
+                                duffy_scp_in('otel_env', 'otel_env', boxname, './')
```
Comment (ekohl) [inline centos.org/pipelines/lib/foremanDuffyJob.groovy:97]: Not for this PR, but I was looking at this change myself and wondered about using [sshGet](https://www.jenkins.io/doc/pipeline/steps/ssh-steps/#sshget-ssh-steps-sshget-get-a-filedirectory-from-remote-node), [sshPut](https://www.jenkins.io/doc/pipeline/steps/ssh-steps/#sshput-ssh-steps-sshput-put-a-filedirectory-on-remote-node) and [sshCommand](https://www.jenkins.io/doc/pipeline/steps/ssh-steps/#sshcommand-ssh-steps-sshcommand-execute-command-on-remote-node) instead. Have you thought about that?

Comment (evgeni) [reply centos.org/pipelines/lib/foremanDuffyJob.groovy:97]: I did not know about that, so, no. But certainly something to look at.

**Review by ekohl — APPROVED** [review]
I didn't look at it too carefully. From a quick reading this looks to be correct.

File: centos.org/ansible/fetch_debug_files.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   tasks:
     - name: 'Find any files from debug'
       find:
-        paths: "/tmp/debug-{{ pipeline_type }}-{{ pipeline_version }}-{{ pipeline_os }}"
+        paths: "/tmp/debug-{{ pipeline_type }}-{{ pipeline_version }}-{{ pipeline_os }}-{{ pipeline_action }}"
```
Comment (ekohl) [inline centos.org/ansible/fetch_debug_files.yml:6]: Just for my own sanity: this expands to `/tmp/debug-foreman-nightly-el8-install`.

File: centos.org/pipelines/lib/foremanDuffyJob.groovy [hunk 3]
```diff
@@ -66,22 +81,23 @@ pipeline {
...
 
                     writeFile(file: 'otel_env', text: otel_env)
-                    duffy_scp_in('otel_env', 'otel_env', 'duffy_box', './')
-                    sh(script: 'rm -rf otel_env', label: 'remove otel_env file')
 
                     def branches = [:]
                     pipelines.each { action, oses ->
                         oses.each { os ->
                             def name = "${os}-${action}"
+                            def username = "pipe-${name}"
```
Comment (ekohl) [inline centos.org/pipelines/lib/foremanDuffyJob.groovy:89]: On line 120 you have this:
```suggestion
                            def username = "pipe-${name}-${action}"
```

Comment (evgeni) [reply centos.org/pipelines/lib/foremanDuffyJob.groovy:89]: No, there it is `pipe-${os}-${action}` (as `$name` is different there)
but I agree it's inconsistent.

**Review by ekohl — APPROVED** [review]

**Pull request merged**